### PR TITLE
fix: m949, ensure site with coordinates of 0,0 shows up as a marker on the site form map

### PR DESCRIPTION
**Pull request by mmnoo**

### CODE REVIEW

**Comment by mmnoo** [comment]
hold off on this! I found more weirdness (with the save button)

File: src/components/pages/Site/Site.js [hunk 1]
```diff
@@ -380,7 +380,7 @@ const Site = ({ isNewSite }) => {
         errors.latitude = [{ code: language.error.formValidation.latitude, id: 'Invalid Latitude' }]
       }
 
-      if (!values.longitude && values.latitude !== 0) {
+      if (!values.longitude && values.longitude !== 0) {
```
Comment (mmnoo) [inline src/components/pages/Site/Site.js:383]: oof

**Comment by mmnoo** [comment]
k, this is ready for review and qa. I misunderstood the save button requirements before.

**Comment by mmnoo** [comment]
This straggler is ready for PR @saanobhaai (I deleted the outdated comment that put it on hold)

**Comment by saanobhaai** [comment]
[New site doesnt allow lat long of 0, 0](https://trello.com/c/EZSRI3Ux/949-new-site-doesnt-allow-lat-long-of-0-0)

**Comment by saanobhaai** [comment]
[New site doesnt allow lat long of 0, 0](https://trello.com/c/EZSRI3Ux/949-new-site-doesnt-allow-lat-long-of-0-0)

**Review by saanobhaai — APPROVED** [review]

**Pull request merged**